MultiPartyExample/MultiPartyViewController.swift

```diff
@@ -8,6 +8,99 @@
 import UIKit
 import TwilioVideo
 
+struct CaptureDeviceUtils {
+
+    static let kOneToOneFrameRate = UInt(24)
+    static let kOneToOneVideoBitrate = UInt(1120) * 1024
+    static let kMultipartyFrameRate = UInt(15)
+    static let kMultipartyVideoBitrate = UInt(600) * 1024
+    static let kSimulcastVideoBitrate = UInt(1800) * 1024
+    static let kOneToOneVideoDimensions = CMVideoDimensions(width: 640, height: 480)
+    static let kMultipartyVideoDimensions = CMVideoDimensions(width: 480, height: 360)
+    static let kSimulcastVideoDimensions = CMVideoDimensions(width: 1280, height: 960)
```

Comment:
Would be nice to bundle these as distinct configurations, which include the 3 constants.